Supervision Work/src/uk/ac/cam/ap886/oopjava/supervisionProjects/Hangman/Game.java

```diff
@@ -0,0 +1,115 @@
+package uk.ac.cam.ap886.oopjava.supervisionProjects.Hangman;
+
+import java.util.ArrayList;
+
+public class Game {
+	protected 	Dictionary 	dictionary;		
+	protected 	Player 		player;		  	
+	protected 	char[] 		etalon;			
```

Comment:
If I have public setters, then anyone can inject the answer they need. I don't want that. But I still want to be able to use a different implementation. I could have protected setters, which is just extra overhead.